### ISSUE TITLE
systemd-nspawn doesn't print error messages

### ISSUE THREAD

**Issue by iaguis** [systemd/systemd#207]
Here I try to bind-mount a directory that doesn't exist. It doesn't print an error message, it just exists with status 1:

```
iaguis@locke-work: ~/temp
> sudo systemd-nspawn --bind /idontexist:/tmp -bD ~/temp/nspawn/arch-tree 
Spawning container arch-tree on /home/iaguis/temp/nspawn/arch-tree.
Press ^] three times within 1s to kill container.
iaguis@locke-work: ~/temp/rkt 
> echo $?
1
```

However, if I redirect stdout to a file it does print the error:

```
iaguis@locke-work: ~/temp/rkt 
> sudo systemd-nspawn --bind /idontexist:/tmp -bD ~/temp/nspawn/arch-tree  > /tmp/kk.txt
Spawning container arch-tree on /home/iaguis/temp/nspawn/arch-tree.
Press ^] three times within 1s to kill container.
/etc/localtime is not a symlink, not updating container timezone.
Failed to stat /idontexist: No such file or directory
```

[systemd-nspawn closes the stdout file descriptor when running interactively](https://github.com/systemd/systemd/blob/0c33502bed83f2775d77c992fe621ec85e75ab2a/src/nspawn/nspawn.c#L4267) so after that point, no error messages are printed on screen.

Is this intended behavior?

**Comment by poettering** [comment]
In the non-redirectied case the TTY handling currently simply doesn't flush out whatever is queued before exiting nspawn. and that's certainly a bug.

In the redirected case we just connect the real fd with the processes we fork, which means our borked tty handling doesn't matter. 

This is something to fix the the ptyfwd stuff...

**Comment by iaguis** [comment]
Fixed by #272. Thanks!
